debugging/debugging.md

```diff
@@ -45,7 +45,7 @@
 - If the red squiggly underline is under something that's clearly correct, like `color: red;` in a CSS file, then first check if it really is correct.
 - The phrase "pattern matching" and why it's important: use other code snippets to remind yourself of correct syntax.
```

Comment:
Can you actually describe pattern matching here

Reply:
@vsharma20v , please rephrase this pattern-matching line and add an example as @alodahl said because right now, it is not very clear what "pattern matching" you are talking about.

Reply:
@vsharma20v , I will resolve this conversation when @alodahl will approve the change you made.